homeassistant/components/nordpool/quality_scale.yaml

```diff
@@ -0,0 +1,101 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
```

Comment:
isn't that `done` then?

Reply:
Fair point, not entirely obvious exempt vs. done in many cases.